rpcs3/Emu/RSX/Overlays/overlay_edit_text.cpp

```diff
@@ -111,4 +195,4 @@ namespace rsx
 			return compiled_resources;
 		}
 	} // namespace overlays
-} // namespace rsx
+}  // namespace rsx
```

Comment:
unnecessary space

Reply:
Yeah, I know. Was going to eliminate it in another PR. But since you've found a bug, will try to fix it and include elimination there.